eclass/java-pkg-simple.eclass

```diff
@@ -417,6 +417,7 @@ java-pkg-simple_src_compile() {
 	# package
 	local jar_args
 	if [[ -e ${classes}/META-INF/MANIFEST.MF ]]; then
+		sed '/Created-By: /d' -i ${classes}/META-INF/MANIFEST.MF
```

Comment:
i found an info that headers in the manifest file, though should use first letters capital, are case insensitive, so we should make the regex case insensitive, to be sure it catches even some corner cases.

Reply:
> i found an info that headers in the manifest file, though should use first letters capital, are case insensitive, so we should make the regex case insensitive, to be sure it catches even some corner cases.

`/Id` does the trick.